lib/quby/questionnaires/dsl/questionnaire_builder.rb

```diff
@@ -54,6 +54,17 @@ def sbg_key(sbg_key)
           @questionnaire.sbg_key = sbg_key
         end
 
+        def sbg_domain(sbg_code, from: nil, till: nil, outcome: nil, sbg_key: nil, primary: false)
```

Comment:
Outcome is toch de Meetinstrument Code? @veermans waarom noemen we dat outcome?  In de xls is die voor elke lijst aangegeven. In de tabel staat elke lijst ook steeds onder een specifiek Meetinstrument.